client/blocks/plan-thank-you-card/docs/example.jsx

```diff
@@ -0,0 +1,28 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import sitesList from 'lib/sites-list';
+import PlanThankYouCard from 'blocks/plan-thank-you-card';
+
+const sites = sitesList();
+
+const PlanThankYouCardExample = React.createClass( {
+	displayName: 'PlanThankYouCard',
+	render() {
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/blocks/plan-thank-you-card">Plan Thank You Card</a>
+				</h2>
+				<PlanThankYouCard selectedSite={ sites.getPrimary() } />
```

Comment:
Parent component was using that, so I rolled with it. I switched to passing just `siteId` and querying redux state inside the card.